### PR TITLE
Adjust Text Size of Module Buttons

### LINKED ISSUE #86: Adjust sizes of Text
Text in Courses should be larger than the text in the legend <3

### PR DESCRIPTION
**Pull request by noobiba**
Resolves  #86 Adjust sizes of Text.
@sarahkami @lauralindal @lenakoester Could you please review. Thank you! :-)

### CODE REVIEW

File: src/Module.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ class Module extends React.Component {
     return (
       <div key={this.props.module.title} className="semester">
           <div className={ModuleClasses} onClick={this.toggleModule.bind(this)}>{this.props.module.title} <br/>
-          {this.props.userModule.status}</div>
+          <span style={{fontStyle: 'italic', fontWeight: '400'}}>{this.props.userModule.status}</span></div>
```
Comment (lauralindal) [inline src/Module.js]: I'm no CSS/HTML expert, but is it okay to do this inline? @bastianalbers

**Review by lauralindal — CHANGES_REQUESTED** [review]

**Comment by noobiba** [comment]
I guess I understood why it didn´t work out when I tried it via css class style. Now I inserted a class instead of inline css. It should work now.

**Comment by noobiba** [comment]
@sarahkami @lauralindal @lenakoester  Please review again. Thank you.

**Comment by bastianalbers** [comment]
While it is OK to write inline styles in react, i personally would try to not do it. Also because your styles all live in the css files, having inline styles adds another source of truth for the styling. So putting that into a class like you did is exactly the right, or, errm, lets say preferred way for me. 👍

**Review by lauralindal — APPROVED** [review]

**Pull request merged**